release-notes-8.0.0.md

```diff
@@ -18,5 +15,12 @@ ${version-number}
 #### Breaking Changes
 - **US749035**: Classic queues and priority queues are deprecated, priority queues can still be created however it is no
   longer possible to publish messages with a priority.
-  When using `cfg~caf~worker~RabbitConfiguration.js`, the type of queue created by workers can be controlled by the ENV
-  CAF_RABBITMQ_QUEUE_TYPE, this currently defaults to 'quorum', but will be removed in a future release.
+  When using `cfg~caf~worker~RabbitWorkerQueueConfiguration.js`, the type of queue created by workers can be controlled 
+  by the ENV CAF_RABBITMQ_QUEUE_TYPE, this currently defaults to 'quorum', but will be removed in a future release.
+- **US857114**: Introduced configurable Rabbit MQ protocol so that, if desired, Rabbit MQ communication can be TLS 
+  enabled. When using `cfg~caf~worker~RabbitConfiguration`, the rabbit protocol used by services can be configured using 
+  the CAF_RABBITMQ_PROTOCOL environment variable. The default value is 'amqp'.
+  Consumers using the util-rabbitmq module will need to handle additional exceptions when creating a Rabbit connection
+  through `RabbitUtil.java`. If creating a Rabbit connection using default configuration, where previously host, port, 
```

Comment:
Reword
If creating a Rabbit connection using default configuration, where previously host, port, user and pass were provided, protocol will now also be required.

To
RabbitUtil.createRabbitConnection now requires a protocol argument.